jme3-ios/ios-data/templates/project/jme-ios/jmeAppDelegate.m

```diff
@@ -273,15 +273,19 @@ - (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event {
     NSLog(@"touchesBegan");
     JNIEnv* e = getEnv(self.vm);
     if (e) {
-    	UITouch *touch = [touches anyObject];
-    	CGPoint position = [touch locationInView: nil];
-        float scale = _glview.contentScaleFactor;
-        // NOTE: cast to jlong is required, otherwise JmeAppHarness receives non-sense values
-        (*e)->CallVoidMethod(e, self.harness, self.injectTouchBegin, 0, (jlong) touch.timestamp, position.x * scale, position.y * scale);
-        if ((*e)->ExceptionCheck(e)) {
-            NSLog(@"Could not invoke iOS Harness injectTouchBegin");
-            (*e)->ExceptionDescribe(e);
-            (*e)->ExceptionClear(e);
+        NSEnumerator *enumerator = [touches objectEnumerator];
+        UITouch *touch;
+        while (touch = [enumerator nextObject]) {
+            CGPoint position = [touch locationInView: nil];
+            float scale = _glview.contentScaleFactor;
+            //NSLog(@"multipleTouchEnabled=%d", _window.rootViewController.view.multipleTouchEnabled); // true
+            // NOTE: cast to jlong is required, otherwise JmeAppHarness receives non-sense values
```

Comment:
It is unclear what this logging statement should be/why it is commented.
The second comment should probably be copied to every related CallVoidMethod statement